udata/tests/organization/test_organization_rdf.py

```diff
@@ -103,10 +109,12 @@ def test_catalog_pagination(self):
             _external=True,
         )
         DatasetFactory.create_batch(total, organization=origin_org)
+        DataserviceFactory.create_batch(total, organization=origin_org)
 
         # First page
         datasets = Dataset.objects.paginate(1, page_size)
-        catalog = build_org_catalog(origin_org, datasets, format="json")
+        dataservices = Dataservice.objects.paginate(1, page_size)
```

Comment:
Do we need to update this logic in test?

Reply:
ah, yes, indeed, good catch, i'll update the tests 🙏